lib/optimizely.rb

```diff
@@ -523,6 +524,52 @@ def close
       @event_processor.stop! if @event_processor.respond_to?(:stop!)
     end
 
+    def get_optimizely_config # rubocop:disable Naming/AccessorMethodName
```

Comment:
Instead of disabling it inline here. Please put in rubocop_todo.yml. Makes it easier to see all rule exceptions at one place. 
You can even use this utility to auto generate todo file for you. 

> rubocop --auto-gen-config

Reply:
this particular rule can only be disabled for whole file using rubocop_todo which i think is not a good idea. What do you suggest ?

Reply:
I see. I would choose to keep in the todo file. But you may keep it this way.

Reply:
We shouldn't depend our code files on rubocop. Please add the rule in rubocop_todo.yml.

Reply:
Done